Add Teiler and Exporter to the main README.md (#315)
samply/bridgehead: README.md

You can find the profiles for generating FHIR in [Simplifier](https://simplifier.net/bbmri.de/~resources?category=Profile).

### Teiler (Frontend)

> Teiler is the **web-based frontend of the Bridgehead**, providing **access** to its various **embedded, internal, and external services**.   

Reviewer: ```suggestion
Teiler is the web-based frontend of the Bridgehead, providing access to its various internal, and external services and components.   
```

> Its architecture is based on microfrontends, which allows individual modules—potentially backed by dedicated microservices—to be integrated in a straightforward and modular way.    

Reviewer: ```suggestion
To learn how to integrate your custom module into Teiler, please refer to https://github.com/samply/teiler-dashboard.
```

- To activate Teiler, set the following environment variable in your `<PROJECT>.conf` file:  

```bash
ENABLE_TEILER=true
```

### Data Exporter Service

> The Exporter is a dedicated service for **extracting and exporting raw data in tabular formats** such as **Excel, CSV, Opal, JSON, XML**...  

Reviewer: ```suggestion
The Exporter is a dedicated service for extracting and exporting Bridgehead data in (tabular) formats such as Excel, CSV, Opal, JSON, XML, ...  
```

> Its architecture is template-based, allowing users to define **what data** to extract and from **which sources** (e.g., **Blaze**).

Reviewer: ```suggestion
- To enable the Exporter service, set the following environment variable in your `<PROJECT>.conf` file:  

```bash
ENABLE_EXPORTER=true
```
```

#### Data Quality Report

> Additionally, a plugin is available for generating **advanced Excel reports** that include built-in **data analysis**. One such template is **preconfigured** to generate **data quality reports** out of the box.  
> 
> - To enable the Exporter service, set the following environment variable in your `<PROJECT>.conf` file:  
> 

Reviewer: ```suggestion
To assess the quality and plausibility of your imported data, the Reporter component is pre-configured to generate Excel reports with data quality metrics and statistical analyses. Reporter is part of the Exporter and can be enabled by setting the same environment variable in your `<PROJECT>.conf` file:
```

```bash
ENABLE_EXPORTER=true
```

> The **Teiler** frontend includes embedded apps that provide **user-friendly interfaces** for both **exporting** data and generating **quality reports**. For this reason, it is **recommended to enable Teiler together with the Exporter** service to fully utilize these integrated features.  
> - Suggested configuration in your `<PROJECT>.conf` file:

Reviewer: ```suggestion
For convenience, it's recommended to enable the Teiler web frontend alongside the Exporter to access export and quality control features via a web interface: set the following environment varibles in your `<PROJECT>.conf` file:
```

```bash
ENABLE_TEILER=true
ENABLE_EXPORTER=true
```

> 

Reviewer: ```suggestion
```
